app/models/payments/ChargeType.scala

```diff
@@ -399,7 +419,12 @@ object ChargeType extends LoggerUtil {
     VatAAMonthlyInstalLPI,
     VatAAReturnCharge1stLPP,
     VatAAReturnCharge2ndLPP,
-    VatReturn2ndLPP
+    VatReturn2ndLPP,
+    VatErrorCorrectionLPI,
+    VatErrorCorrection1stLPP,
+    VatErrorCorrection2ndLPP,
+    VatErrorCorrection1stLPPLPI,
+    VatErrorCorrection2ndLPPLPI
```

Comment:
Not related to your change here but could you remove the `positiveOrNegativeChargeTypes` val and move the 2 charge types from that list into this big list? I have no idea why that exists, I can't see any logic for it.